### PR TITLE
remove the section from README to avoid confusion

### PR DESCRIPTION
**Pull request by yzhao583**
remove the section from README to avoid confusion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,12 +50,6 @@ yarn dev
 |ExampleFetchComponent.tsx|AppengGHReposComponent.tsx|
 |ExampleFetchComponent.test.tsx|AppengGHReposComponent.test.tsx|
```
Comment (rhkp) [inline README.md:72]: Hi @yzhao583 Thanks. We may need to keep this section as the section with title `Ensure AppengGHReposComponent is now being used` makes use of this to prove the updates have taken place. Please let me know your prefernce.

Comment (yzhao583) [reply README.md:72]: @rhkp I just cloned the repo and followed the README, and went to the "AppengGHReposComponent.tsx" file, and I did not find any email, so I felt confused. In addition, the code in "ExampleFetchComponent.tsx" is just trying to simulate a fetch operation, and it is already replaced by a real API call in "AppengGHReposComponent.tsx", so I think it is not necessary to still keep the related content there. WDYT?

Comment (rhkp) [reply README.md:72]: Okay then let us remove this section.

**Review by rhkp — APPROVED** [review]
lgtm

**Pull request merged**